include/rocksdb/utilities/customizable_util.h

```diff
@@ -68,6 +68,9 @@ static Status NewSharedObject(
     if (config_options.ignore_unsupported_options && status.IsNotSupported()) {
       return Status::OK();
     }
+  } else if (opt_map.empty()) {
```

Comment:
Add the logic change to the comment. Previously, if id is empty, we will return not supported. But now, we will reset the `result` to nullptr. Also, can you explain a little bit why we need this change?